thresholdkey_generator.go

```diff
@@ -6,21 +6,32 @@ import (
 	"math/big"
 )
 
+// Generates a threshold Paillier key with an algorithm based on [DJN 10],
+// section 5.1, "Key generation".
+//
+// Bear in mind that the algorithm assumes an existence of a trusted dealer
```

Comment:
Will this implementation still work for us if it assumed a trusted dealer?

Reply:
We'll need to build on a top of it. We need to add a distributed Key-Gen algorithm but the rest of the code (signing, verification etc) should remain unchanged. 

I am evaluating possible T-GenKey approaches now.